aten/src/ATen/EmptyTensor.cpp

```diff
@@ -16,25 +16,27 @@ c10::Allocator* GetCPUAllocatorMaybePinned(bool pin_memory) {
     // NB: This is not quite right, if you somehow had both CUDA and PrivateUse1 initialized
     // in the same PyTorch build, you would ONLY ever get the CUDA pinned memory allocator.
     // To properly support this, see https://github.com/pytorch/pytorch/issues/14560
+
+    std::optional<c10::DeviceType> opt_device_type = std::nullopt;
+    // Preserved for BC, CUDA first
```

Comment:
Can you please clarify that this is CUDA vs PrivateUse1 ordering in particular.

Reply:
Done, please help to take a loot at it, thank you. :D